### PR TITLE
Add Madison PHP to the list

### PR DESCRIPTION
**Pull request by otte-ford**
We did a TestFest event at Madison PHP Conference 2017 on September 22, 2017.

### CODE REVIEW

File: docs/_data/groups/2017.yml [hunk 1]
```diff
@@ -80,6 +80,19 @@
     - name: Eric Poe
       email: eric.poe@gmail.com
 
+- name: Madison PHP
+  location: Madison, WI, USA
+  website: http://www.madisonphpconference.com
+  meetup: https://www.madisonphp.com
```
Comment (ramsey) [inline docs/_data/groups/2017.yml]: This URL is timing out for me, but when I request it without "https," it properly redirects to your group on Meetup.com. Can you check on the URL and update the pull request, if needed? Thanks!

**Review by ramsey — CHANGES_REQUESTED** [review]

**Comment by ramsey** [comment]
Not sure if you saw my request for changes on this, so just double-checking. :smile:

File: docs/_data/groups/2017.yml (hunk 1 above)
Comment (otte-ford) [reply docs/_data/groups/2017.yml]: Hi Ben, oh no - I didn’t - thanks for getting back to me! You’re right the Madison PHP should be without the s (so http://www.madisonphp.com , which then redirects to Meetup, which has https on it). I - pushed the change, which seems to have updated the pull request - let me know if you need anything else. Thanks for organizing this!

**Review by ramsey — APPROVED** [review]
Thanks, @otte-ford! I just noticed that this is your first pull request on GitHub. Congratulations!

**Pull request merged**